# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libacpi/install.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix install target to remove existing destination files while creating
symlinks for the shared library. 

--- libacpi-0.2/Makefile.orig	2010-07-27 17:07:09.108256069 +0200
+++ libacpi-0.2/Makefile	2010-07-27 17:07:26.412027931 +0200
@@ -48,7 +48,7 @@
 	@chmod 644 ${DESTDIR}${PREFIX}/lib/libacpi.a
 	@cp -f ${SONAME} ${DESTDIR}${PREFIX}/lib/
 	@chmod 644 ${DESTDIR}${PREFIX}/lib/${SONAME}
-	@ln -s ${SONAME} ${DESTDIR}${PREFIX}/lib/libacpi.so
+	@ln -sf ${SONAME} ${DESTDIR}${PREFIX}/lib/libacpi.so
 	@echo installing test-libacpi to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@cp -f test-libacpi ${DESTDIR}${PREFIX}/bin
